pgmq-extension/sql/pgmq--1.5.1--1.5.2.sql

```diff
@@ -102,3 +102,155 @@ BEGIN
     RETURN TRUE;
 END;
 $$ LANGUAGE plpgsql;
+
+DROP FUNCTION pgmq.create_non_partitioned(TEXT);
+CREATE FUNCTION pgmq.create_non_partitioned(queue_name TEXT)
+RETURNS void AS $$
+DECLARE
+  qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+  qtable_seq TEXT := qtable || '_msg_id_seq';
+  atable TEXT := pgmq.format_table_name(queue_name, 'a');
+  lock_key BIGINT;
+BEGIN
+  PERFORM pgmq.validate_queue_name(queue_name);
+  PERFORM pg_advisory_xact_lock(hashtext('pgmq.create_non_partitioned' || queue_name)););
```

Comment:
Should we get the same lock on drop?

Reply:
yeah that is a good idea. i'll push that into this PR too.

Reply:
pushed that in https://github.com/tembo-io/pgmq/pull/393/commits/b5a0a662c953ecb3f1ddfa5e2018b1c672e4a812